modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogColumnCollation.java

```diff
@@ -27,6 +27,8 @@ public enum CatalogColumnCollation {
     DESC_NULLS_FIRST(false, true),
     DESC_NULLS_LAST(false, false);
 
+    private static final int ASC_BIT_POS = 0;
+    private static final int NULLS_FIRST_BIT_POS = 1;
```

Comment:
```suggestion
    private static final int ASC_BIT_POS = 1;
    private static final int NULLS_FIRST_BIT_POS = 1 << 1;
```

Reply:
:thinking: :thinking:  reverted last changes and renamed to ASC_FLAG_**BITMASK**...